src/whylogs/app/session.py

```diff
@@ -8,6 +8,7 @@
 from uuid import uuid4
 
 import pandas as pd
+from tqdm.auto import tqdm
```

Comment:
we shouldn't depend on `tqdm` as a hard requirement. It's a cosmetic thing, and in this case, Jupyter notebook so it can be an extra dependency with option import (`try except...` pattern for import)

I know we introduced tqdm in `logger.py` but this is not a good pattern and we should stop doing it

Reply:
Talked offline, we will depend on it but it will default to off.